zetaclient/chains/bitcoin/tx_script.go

```diff
@@ -337,11 +337,11 @@ func decodeInscriptionPayload(t *txscript.ScriptTokenizer) ([]byte, error) {
 // OP_PUSHBYTES_32 <32 bytes> OP_CHECKSIG <Content>
 func checkInscriptionEnvelope(t *txscript.ScriptTokenizer) error {
 	if !t.Next() || t.Opcode() != txscript.OP_DATA_32 {
-		return fmt.Errorf("cannot obtain public key bytes op %d or err %s", t.Opcode(), t.Err())
+		return fmt.Errorf("public key not found")
```

Comment:
Isn't some context we can still keep in the error?

Reply:
Appended error (if any) to the log print: https://github.com/zeta-chain/node/pull/3155/commits/80d275a5b6cb9b9b8d9ae85a346b85f1bcc4208d